app/services/publishers/dfe_sign_in/big_query_export/base.rb

```diff
@@ -0,0 +1,26 @@
+require "dfe_sign_in/api"
+require "google/cloud/bigquery"
+
+module Publishers::DfeSignIn::BigQueryExport
+  class Base
+    include DfeSignIn::API
+    include Publishers::DfeSignIn::Parsing
```

Comment:
Using composition through modules, we can combine library code (API calls) with the domain-related parsing module.